src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/full.xml

```diff
@@ -6,7 +6,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd
                         http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
 
-    <framework:config secret="s3cr3t" ide="file%%link%%format" default-locale="fr" http-method-override="false">
+    <framework:config secret="s3cr3t" ide="file%%link%%format" default-locale="fr" available-locales="fr" http-method-override="false">
```

Comment:
how does this work when there are several of them?

Reply:
you're right, my mistake here.